tsconfig.base.json

```diff
@@ -9,6 +9,7 @@
 		// See also https://aka.ms/tsconfig/module
 		"module": "nodenext",
 		"target": "esnext",
+		"jsx": "react-native",
```

Comment:
no sure this needs to be in the base config? If it just applies to react native add it in the tsconfig in the /mobile package